ci/tests/tracing/scenarios/tyk_test_with_response_mw.yml

```diff
@@ -0,0 +1,17 @@
+type: Test
+spec:
+  id: 14JFah64R
+  name: response mw
+  description: test the execution of the response mw
+  trigger:
+    type: http
+    httpRequest:
+      method: GET
+      url: tyk:8080/test-response-mw/ip
+      headers:
+        - key: Content-Type
+          value: application/json
+  specs:
```

Comment:
can you check here if this span contains the actual parent span? to check the issue we discussed earlier in the Slack channel - I don't think it'd be so difficult with tracetest

Reply:
makes the whole sense, done!